packages/stylesheets/base/_colors.scss

```diff
@@ -0,0 +1,10 @@
+$primary-color: var(--nge-primary-color, #fafafa);
+$primary-foreground-color: var(--nge-primary-foreground-color, #18181b);
+
+$secondary-color: var(--nge-secondary-color, #27272a);
+// $secondary-foreground-color: var(--nge-secondary-foreground-color, );
```

Comment:
I think you forgot to finish this

Reply:
No, I didn't have to use a secondary color foreground so we can remove this